src/child/actions/window.js

```diff
@@ -75,6 +82,54 @@ function updatingOptionsError() {
     };
 }
 
+export function resizeToMin() {
```

Comment:
Can we combine this with the `minimise` function?
If the existing minimise function is used elsewhere, its probably worth just using the actions that is dispatched from the success of this promise, since we're not minimised unless this promise succeeds.
I'd rename this to `minimising` as well, then `minimiseSuccess` and `minimiseError`, or resizing, whatever you think.

Same applies to the maximise I think?